models/mqtt.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "golang.zx2c4.com/wireguard/wgctrl/wgtypes"
+
+type PeerUpdate struct {
```

Comment:
comments required for public types, funcs and variables

Reply:
added comments